### ISSUE TITLE
Parent component bindings not passed to child component correctly

### ISSUE THREAD

**Issue by jodymgustafson** [brianmhunt/knockout-secure-binding#44]
When you have a parent component and you pass an observable parameter to a child component, the value is being unwrapped.

For example:
```html
    <parent params="value: appvalue"></parent>
    
    <script type="text/html" id="parent">
      <child params="value: parentvalue"></child>
    </script>

    <script type="text/html" id="child">
      <input type="text" data-bind="value: childvalue"></input>
    </script>
```
*See all the code in plunker: https://plnkr.co/edit/EzpJD3yXd01aqPbuOq1X*

Here appvalue is a ko.observable. It gets passed into the "parent" component and is still a ko.observable object. Then the "parent" component passes it into the "child" component. Inside the "child" component the value has been unwrapped. It is no longer an instance of ko.observable, but a string.

If you use the default knockout binding it works correctly. Is there some reason this doesn't work in ksb or is it a bug?

**Comment by brianmhunt** [comment]
Thanks for the detailed report, @jodymgustafson 

In this scenario I believe KSB should behave exactly as KO proper.

I'll write up a test isolating the issue and make sure it behaves as expected, hopefully shortly. :)

**Comment by brianmhunt** [comment]
@jodymgustafson – This should hopefully be resolved now in 0.5.4 (just published).  Please re-open if this still appears to be an issue.

Sorry for the issue, and thanks for the report.

Cheers.

**Comment by jodymgustafson** [comment]
Awesome, I just tried it and it seems to be working as expected. One thing though. It looks like your minifier went haywire because the min version of the file has an error it. Says it's missing a closing curly brace.

**Comment by brianmhunt** [comment]
@jodymgustafson Thanks, appreciate the note re. the minifier.  Will have a look ASAP.

**Comment by brianmhunt** [comment]
Minified version should now be fixed in 0.5.5, but let me know if you encounter any further issues.

Cheers.

**Comment by jodymgustafson** [comment]
That works much better. Thanks so much for creating ksb and for fixing this so quickly. This is the last thing I need to get my app working.
